workflows/phaseimpute/tests/test_all.nf.test

```diff
@@ -0,0 +1,217 @@
+                outdir = "$outputDir"
+                max_cpus = 4
+                max_memory = '5.GB'
+            }
+        }
+
+        then {
+            assertAll(
+                { assert workflow.success },
+                { assert snapshot(
+                    path("${outputDir}/imputation/")
+                        .list()
+                        .collect { getRecursiveFileNames(it, outputDir) }
+                        .flatten(),
+                    path("$outputDir/imputation/glimpse2/concat/NA12878_glimpse2.vcf.gz").linesGzip.size()
+                ).match()
+                }
+            )
+        }
+    }*/
+
+    test("Check test_quilt") {
+        tag "test_quilt"
+        config "../../../conf/test_quilt.config"
+        when {
+            params {
+                pipelines_testdata_base_path = 'https://raw.githubusercontent.com/nf-core/test-datasets/phaseimpute/data/'
+                outdir = "$outputDir"
+                max_cpus = 4
+                max_memory = '5.GB'
+            }
+        }
+
+        then {
+            assertAll(
+                { assert workflow.success },
+                { assert snapshot(
+                    path("${outputDir}/imputation/")
+                        .list()
+                        .collect { getRecursiveFileNames(it, outputDir) }
+                        .flatten(),
+                    path("$outputDir/imputation/quilt/concat/NA12878_quilt.vcf.gz").linesGzip.size()
+                ).match()
+                }
+            )
+        }
+
+    }
+
+    test("Check test_stitch") {
+        tag "test_stitch"
+        config "../../../conf/test_stitch.config"
+        when {
+            params {
+                pipelines_testdata_base_path = 'https://raw.githubusercontent.com/nf-core/test-datasets/phaseimpute/data/'
+                outdir = "$outputDir"
+                max_cpus = 4
+                max_memory = '5.GB'
+            }
+        }
+
+        then {
+            assertAll(
+                { assert workflow.success },
+                { assert snapshot(
+                    path("${outputDir}/imputation/")
+                        .list()
+                        .collect { getRecursiveFileNames(it, outputDir) }
+                        .flatten(),
+                    path("$outputDir/imputation/stitch/concat/all_samples_stitch.vcf.gz").linesGzip.size()
+                ).match()
+                }
+            )
+        }
+
+    }
+
+    test("Check test_sim") {
+        tag "test_sim"
+        config "../../../conf/test_sim.config"
+        when {
+            params {
+                pipelines_testdata_base_path = 'https://raw.githubusercontent.com/nf-core/test-datasets/phaseimpute/data/'
+                outdir = "$outputDir"
+                max_cpus = 4
+                max_memory = '5.GB'
+            }
+        }
+
+        then {
+            assertAll(
```

Comment:
Can we assert if the results before and after simulation are different? For example, see the number of lines of the original VCF vs. the downsized VCF.